packages/fxa-auth-server/lib/routes/security-events.js

```diff
@@ -0,0 +1,42 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+module.exports = (log, db, config) => {
+  return [
+    {
+      method: 'GET',
+      path: '/securityEvents',
+      options: {
+        auth: {
+          strategy: 'sessionToken',
+        },
+      },
+      handler: async function(request) {
+        log.begin('SecurityEvents', request);
```

Comment:
To help differentiate the routes in the logs, lets use `SecurityEventsGet` and `SecurityEventsDelete`. We did something similar for recovery keys.